### DISCUSSION TITLE
Media Manager for `VIDEO` or just for .mp4

### DISCUSSION THREAD

**Discussion by dangerstudios** [directus/directus#15996] [Feature Requests]
Hello,

I think essentially what is needed is a media manager for video? with ffmpeg transcoding, ffprobe meta data, etc...

Currently it is not possible to manage video in a comprehensive manner. Nor is it possible to create a collection that displays videos

Cheers

**Comment by azrikahar** [comment]
For ffprobe in particular, it is being used in the open PR #6865 👍 

Having a display for video would be great indeed. It could be a new standalone Video display like the current Image display, or perhaps they can be combined as "media" display, similar to the `<file-preview>` component in file library preview 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by MMTHD** [comment]
I work with Directus and a large dataset of videos, and I can say that the experience in Directus is not what I had hoped for. Previews and thumbnails are not visible, so you always have to look into another system to make sure you have selected the right media file.

**Reply by dangerstudios** [reply]
Ah yeah I was wondering about this, so you wouldn't recommend? been using headless wordpress(MySQL/graphQL) as a backend for a large quantity of video management with ffmpeg encoding etc, works well enough

**Reply by MMTHD** [reply]
Since I cannot do without a data management system like Directus due to the complexity of my project and my preference for Directus, I use it.  And if the interfaces were not in Vue but rather React/Stencil/TSX, there would also be a larger marketplace and probably some plugins for better media integration. (Although I believe that such things should be part of the core functionality of a DMS). In Directus, I don't lack anything fundamentally, so before starting my project, I actually expected to have a better video management experience (at least similar to that of images). A simple video preview in the form of a thumbnail would have been sufficient for me, instead of treating the file format as if it were an unknown media file. @rijkvanzanten From my perspective, this post doesn't need any votes to be addressed with priority because there is no room for discussion. By the way, I prefer Directus precisely because I like its UI concept and there are hardly any disruptions in the flow of regular content management.

A simple case speaks louder than 15 votes:

One wants to use Directus to manage videos on their own website.
There are more than 5000 videos in the media pool.
When creating a post/page/WTF, one wants to select a suitable video.
A list of 5000 paginated videos opens without any preview or thumbnail. If the filenames are well chosen, one can manage with the filter, but even then, one has to switch to another platform/OS to find out the filename based on the preview/thumbnail in advance, in order to finally create an entry. It is a DISRUPTION IN THE WORKFLOW for something that, mind you, is a simple matter that is not as rare as one might think :)

**Reply by br41nslug** [reply]
This feature request is almost a year old and from before the restructure. Have you tried making a new Feature Request for your suggestion? There are plenty of implementation details to discuss on that topic. A simple case does not speak louder than 15 votes but a desired case should have no issue getting those 15 upvotes 😄 (fyi if it was trivial to implement it would already have been)
